example/bufferCopy/src/bufferCopy.cpp

```diff
@@ -51,16 +47,10 @@ struct FillBufferKernel
     template<typename TAcc, typename MdSpan>
     ALPAKA_FN_ACC auto operator()(TAcc const& acc, MdSpan data) const -> void
     {
-        using Vec = alpaka::Vec<alpaka::Dim<TAcc>, alpaka::Idx<TAcc>>;
-
-        auto const idx = alpaka::getIdx<alpaka::Grid, alpaka::Threads>(acc);
-        auto const gridSize = alpaka::getWorkDiv<alpaka::Grid, alpaka::Threads>(acc);
-
-        for(size_t z = idx[0]; z < data.extent(0); z += gridSize[0])
-            for(size_t y = idx[1]; y < data.extent(1); y += gridSize[1])
-                for(size_t x = idx[2]; x < data.extent(2); x += gridSize[2])
-                    data(z, y, x)
-                        = alpaka::mapIdx<1u>(Vec{z, y, x}, Vec{data.extent(0), data.extent(1), data.extent(2)})[0];
+        // Use a single 3-dimensional loop
+        for(auto idx : alpaka::uniformElementsND(acc, alpaka::Vec{data.extent(0), data.extent(1), data.extent(2)}))
+            data(idx.z, idx.y, idx.x) // equivalent to data(idx[0], idx[1], idx[2])
```

Comment:
should be `idx.z()` you forgot `()` for all indizies

Reply:
ops :-p